server/lyft/aws/sqs/worker.go

```diff
@@ -0,0 +1,107 @@
+package sqs
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/sqs"
+	"github.com/aws/aws-sdk-go-v2/service/sqs/types"
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/uber-go/tally"
+	"sync"
+)
+
+const (
+	ProcessMessageMetricName = "process_msg"
+	ReceiveMessageMetricName = "receive_msg"
+	DeleteMessageMetricName  = "delete_msg"
```

Comment:
sorry to be pedantic about these, but if we can keep these one word it just looks a lot cleaner.

ie. 

`atlantis.sqs.msg.receive` looks better than `atlantis.sqs.process_msg` and allows for a more flexible hierarchy.